.github/workflows/build-as-image.yml

```diff
@@ -0,0 +1,84 @@
+name: Build CoCo AS/RVPS Image
+
+on:
+  workflow_call:
+    inputs:
+      build_option:
+        description: 'Build option for the image'
+        type: string
+
+jobs:
+  build_as_image:
+    strategy:
+      fail-fast: false
+      matrix:
+        instance:
+          - ubuntu-latest
+          - s390x
+        tag:
+          - coco-as-grpc
+          - coco-as-restful
+          - rvps
+        verifier:
+          - all-verifier
+          - se-verifier
+        include:
```

Comment:
this looks like we're enumerating every combination, doesn't it work like this?

since we have 1:1 mapping from name to docker_file + tag

and we also have a 1:1 mapping from instance to verifier

```yaml
matrix:
  name:
  - gRPC CoCo-AS
  - ...
  instance:
  - ubuntu-latest
  - s390x
  include:
  - name: gRPC CoCo-AS
    docker_file: attestation-service/docker/as-grpc/Dockerfile
    tag: coco-as-grpc
  - name: ...
 - instance: s390x
   verifier: se-verifier
 - instance: ubuntu-latest
   verifier: all-verifier   
```

Reply:
It works like https://github.com/BbolroC/trustee/actions/runs/11839493749

Reply:
And your suggestion does not work as expected because:
- These explicitly added `include` combinations do not negate the `exclude` entries
- This still allows all possible combinations of instance and verifier to remain valid, whereas the current configuration explicitly removes some

Reply:
yeah the idea is that it'll produce the same output, but with less repitition

you wouldn't need an exclude section because "verifier" is not part of the matrix, the only matrix is:

$instance * $name (so 2 * 3 options)

the "include" statement just adds additional attributes to valid combinations

Reply:
since this code is just moved around in this PR we can leave it in place here